### DISCUSSION TITLE
Console login error

### DISCUSSION THREAD

**Discussion by aarcex** [fleetbase/fleetbase#214] [Q&A]
I get "network request failed" error. All containers are running ok
Where/how to start troubleshooting?

**Comment by 0Jerel0** [comment]
same error here

**Reply by nperezv** [reply]
I'm having the same issue. I thought that when finishing installation , would be able to create an account but it redirects to the main login and could not create a new account. Please, help with this!!!

**Comment by nperezv** [comment]
Please, help!

**Comment by tmsmatchproject26** [comment]
This system is trash

**Comment by tmsmatchproject26** [comment]
Y'all should be ashamed of yourself for creating a system that crashes.
